doc/src/explore/devnet.md

```diff
@@ -43,8 +41,8 @@ You should see the path to the command. Otherwise, reinstall.
 
 In addition, to conduct advanced work such as publishing a Move module or making a Move call, also obtain:
 
-1. a [GitHub account](https://github.com/signup) if you don't have one already
-1. the [Sui source code](../build/install.md#source-code); for simplicity, we recommend installing in `~/sui` or using an environment variable
+1. a [GitHub account](https://github.com/signup) if you don't have one already.
```

Comment:
A GH account is not a requirement to clone a repo and build code in my experience.

Reply:
Yes, that is correct, it configures git and has no relationship with Github, Inc., a fully-owned subsidiary of Microsoft. Requiring a registration there would not be ideal.

Reply:
> Yes, that is correct, it configures git and has no relationship with Github, Inc., a fully-owned subsidiary of Microsoft. Requiring a registration there would not be ideal.

Thanks for confirming. Removed.